main.go

```diff
@@ -65,7 +65,7 @@ func getOpts() (Opts, error) {
 	return opts, nil
 }
 
-func run() {
+func Run() {
```

Comment:
Was this just changed so that we can access this function within the test suite? Private functions can still be tested as long as the test file is within the same package (in this case they're both in `package main`) so this can probably be reverted